validation/validation_util.jl

```diff
@@ -61,5 +61,6 @@ function extract_resolution_from_filename(str)
         decimal_str = str_non_zero
     end
 
+    # Handle integer numbers to have consistent floats strings.
```

Comment:
```suggestion
    # Convert integer strings to float strings
```
Maybe a bit easier to understand?